README.md

```diff
@@ -62,22 +63,36 @@ posterior = inference.build_posterior()
 performance in some cases. We recommend using a virtual environment with
 [`conda`](https://docs.conda.io/en/latest/miniconda.html) for an easy setup.
 
-To install `sbi`, follow these steps:
+If `conda` is installed on the system, an environment for installing `sbi` can be created as follows:
```

Comment:
This section required restructuring as otherwise you're interviewing too many different (and not needed) ideas of different environment management tools.

Revision requests welcome of course.